models/staging/ga4/stg_ga4__ecommerce.sql

```diff
@@ -0,0 +1,15 @@
+select  
```

Comment:
@dgitis Can you source this from `stg_ga4__events`? That's where we define event & session keys which will serve as primary keys for events and sessions and may be useful during later transformations.

Reply:
As requested, this will be moved to its own pull request. I'll resolve it here.